lib/LoaderRunner.js

```diff
@@ -253,7 +253,10 @@ exports.runLoaders = function runLoaders(options, callback) {
 	var contextDependencies = [];
 
 	// prepare loader objects
-	loaders = loaders.map(createLoaderObject);
+	// loaders = loaders.map(createLoaderObject);
+	for(var loader in loaders) {
+		loaders[loader] = createLoaderObject(loaders[loader]);
+	}
```

Comment:
Was this part of the commit an accident? I just spent a long time hunting down a problem I ran into because I accidentally forked out from master (with this commit) instead of the latest tag.

The error I got was "Error: request should be a string or object with loader and object" when attempting to load extract-text-webpack-plugin.